src/Common/src/System/IO/PathInternal.Unix.cs

```diff
@@ -23,6 +24,11 @@ internal static int GetRootLength(string path)
             return path.Length > 0 && IsDirectorySeparator(path[0]) ? 1 : 0;
         }
 
+        internal static int GetRootLength(ReadOnlySpan<char> path)
```

Comment:
Now that string is implicitly castable to `ReadOnlySpan<char>`, do we still need the string overload?  Is there a perf impact to just having the `ReadOnlySpan<char>` variant?

Reply:
I'm sure it isn't measurable so I'll remove it. I hadn't realized we'd actually made that change.